Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.yml

```diff
@@ -0,0 +1,137 @@
+  name: proxy
+  required: false
+  type: 8
+description: FireMon Security Manager delivers comprehensive rule lifecycle management
+  to help you manage and automate every stage of the change management process. Workflows
+  can be customized and automated to conform to your security goals and standards,
+  with tools at your disposal to evolve policy and protection over time.
+display: FireMon Security Manager - CORTEX XSOAR
+name: FireMonSecurityManager
+script:
+  commands:
+  - deprecated: false
+    description: Returns authentication token
+    execution: false
+    hidden: false
+    name: user-authentication
+    outputs:
+    - contextPath: token
+      type: Unknown
+  - arguments:
+    - default: false
+      description: Domain Id
+      isArray: false
+      name: domain_id
+      required: true
+      secret: false
+    - default: true
+      defaultValue: Access Req WF
+      description: Workflow Name
+      isArray: false
+      name: workflow_name
+      required: false
+      secret: false
+    - default: false
+      description: Add requirement
+      isArray: true
+      name: requirement
+      required: false
+      secret: false
+    - default: true
+      defaultValue: LOW
+      description: Priority of Policy Planner Ticket
+      isArray: false
+      name: priority
+      required: true
+      secret: false
+    - default: false
+      description: Due Date of Policy Planner Ticket
+      isArray: false
+      name: due_date
+      required: true
+      secret: false
+    deprecated: false
+    description: Creates a ticket in policy planner application
+    execution: false
+    hidden: false
+    name: create-pp-ticket
+  - arguments:
+    - default: false
+      description: Domain ID
+      isArray: false
+      name: domain_id
+      required: false
+      secret: false
+    - default: false
+      description: DeviceGroup ID
+      isArray: false
+      name: device_group_id
+      required: false
+      secret: false
+    - default: false
+      description: Enter comma seperated destination values
+      isArray: false
+      name: destinations
+      required: false
+      secret: false
+    - default: false
+      description: Enter comma seperated source values
+      isArray: false
+      name: sources
+      required: false
+      secret: false
+    - default: false
+      description: Enter comma seperated service values
+      isArray: false
+      name: services
+      required: false
+      secret: false
+    - default: false
+      isArray: false
+      name: action
+      required: true
+      secret: false
+    deprecated: false
+    description: "PCA- Pre-Change Assessment is process of showing impact of created\
+      \ devices changes in early stages before implementing changes to devices. \n\
+      We can check the PCA table in Review stage of Policy planner ticket."
+    execution: false
+    hidden: false
+    name: pca
+  dockerimage: demisto/python3:3.9.4.18682
```

Comment:
I think some command are missing here, where is the get-all-workflow

Reply:
resolving as outdated